TaskModel.json

```diff
@@ -815,14 +815,102 @@
 				"Biete den Gästen eine hochklassige und unvergessliche Fahrt nach %2$s.",
 				"Bringe den VSOE mit bestem Ambiente von %s nach %s."
 			],
-			"stations": [ "XFPO", "XIVNS" ],
+			"objects": [
+				{
+					"stations": [ "XFPO", "XIVNS" ],
+					"pathSuggestion": [ "XFPO", "XFCQ", "XFMEL", "XFLAR", "🇫🇷SIF", "🇫🇷TNN", "XFNUR", "🇫🇷PAI", "XFD", "🇫🇷GLS", "🇨🇭COS", "XSBL", "XSGRN", "XSB", "XSOL", "XSKS", "XSZH", "XSTW", "XSPF", "XSSR", "XSBU", "XAFK", "XAI", "🇦🇹I Z1", "XICIG", "XIFF", "XIVP", "XIVNS" ]
+				}
+			],
 			"neededCapacity": [
 				{ "name": "beds" },
 				{ "name": "bistroseats" }
 			],
 			"group": 0,
-			"service": 1,
-			"pathSuggestion": [ "XFPO", "XFCQ", "XFMEL", "XFLAR", "🇫🇷SIF", "🇫🇷TNN", "XFNUR", "🇫🇷PAI", "XFD", "🇫🇷GLS", "🇨🇭COS", "XSBL", "XSGRN", "XSB", "XSOL", "XSKS", "XSZH", "XSTW", "XSPF", "XSSR", "XSBU", "XAFK", "XAI", "🇦🇹I Z1", "XICIG", "XIFF", "XIVP", "XIVNS" ]
+			"service": 1
+		},
+		{
+			"name": "Orient-Express",
+			"descriptions": [
+				"Fahre auf den Spuren des legendären Orient-Express von %s nach %s.",
+				"Bringe die Gäste auf dieser nostalgischen Route nach %2$s."
+			],
+			"objects": [
+				{
+					"stations": [ "XFPO", "XFCM", "XFN", "XFSTG", "TS", "MH", "XASB", "XAWW", "XMG", "XMBK", "XJST", "XJBC", "XJNI", "XWS", "XQIS" ],
+					"descriptions": [
```

Comment:
Beschreibung doppelt und "objects"-Container obwohl es gar keine andere Vererbung gibt?

Reply:
Das ist für mögliche Erweiterungen 😅
Daher auch mit einer allgemeinen, generischen Beschreibung und einer spezifischeren.
 Vielleicht ließe sich sowas auch dadurch lösen, dass `descriptions` in den `objects` mit den allgemeinen zusammengefasst werden?

Reply:
Bzw. ursprünglich hatte ich den Simplon-Orient-Express davon als Subtask erstellt, hätte dann aber ein verschachteltes `objects` gebraucht

Reply:
Alles klar, hat mich nur etwas gewundert gehabt.